cmd/ncproxy/ncproxy.go

```diff
@@ -499,3 +605,24 @@ func (s *ttrpcService) ConfigureNetworking(ctx context.Context, req *ncproxyttrp
 	}
 	return &ncproxyttrpc.ConfigureNetworkingInternalResponse{}, nil
 }
+
+func modifyEndpoint(ctx context.Context, id string, policies []hcn.EndpointPolicy, requestType hcn.RequestType) error {
+	endpointRequest := hcn.PolicyEndpointRequest{
+		Policies: policies,
+	}
+
+	settingsJSON, err := json.Marshal(endpointRequest)
+	if err != nil {
+		return err
+	}
+
+	requestMessage := &hcn.ModifyEndpointSettingRequest{
+		ResourceType: hcn.EndpointResourceTypePolicy,
+		RequestType:  requestType,
+		Settings:     settingsJSON,
+	}
+
+	log.G(ctx).WithField("request", requestMessage).Info("sending HcnModifyEndpoint request")
```

Comment:
We want to keep this?

Reply:
I'm not really sure. Did you ever need this for debugging purposes?

Reply:
Nope, the hns etw events should be enough. Was this my code :P ?

Reply:
Yes :P

Reply:
and this